### PR TITLE
configure db host and port from from config file

### PR DESCRIPTION
**Pull request by xaniasd**
allow configuring all db connection parameters from config file. Set default values to avoid regression

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/1940)

### CODE REVIEW

File: backend/src/apiserver/client_manager.go [hunk 1]
```diff
@@ -210,8 +210,8 @@ func initMysql(driverName string, initConnectionTimeout time.Duration) string {
 	mysqlConfig := client.CreateMySQLConfig(
 		getStringConfigWithDefault(mysqlUser, "root"),
 		getStringConfigWithDefault(mysqlPassword, ""),
-		getStringConfig(mysqlServiceHost),
-		getStringConfig(mysqlServicePort),
+		getStringConfigWithDefault(mysqlServiceHost, "mysql"),
```
Comment (ajchili) [inline backend/src/apiserver/client_manager.go]: If the config.json is not changed or `getStringConfigWithDefaults` fails and returns "mysql" as the host, will a connection be established?

Comment (xaniasd) [reply backend/src/apiserver/client_manager.go]: yes! with the current setup, the database is exposed through a service called mysql. Kubernetes provisions pods with the environment variable `MYSQL_SERVICE_HOST` and the mysql service ip as value. Hence the current value for `mysqlServiceHost`. The same service can also be accessed as host `mysql` so the default behavior remains unaltered.

**Comment by xaniasd** [comment]
@IronPan can this PR be tested? any changes needed?

**Pull request merged**